packages/create-app/src/run.ts

```diff
@@ -105,11 +105,23 @@ export async function run(
     );
   }
 
+  const ourPackageJsonPath = findUpSync("package.json", {
+    cwd: fileURLToPath(import.meta.url),
+  });
+
+  const ourPackageJsonVersion = ourPackageJsonPath
+    ? JSON.parse(fs.readFileSync(ourPackageJsonPath, "utf-8")).version
+    : undefined;
+
+  const clientVersion = process.env.PACKAGE_CLIENT_VERSION
```

Comment:
`process.env.PACKAGE_CLIENT_VERSION` is supposed to be replaced during the build process but the test process does not do that. We can try to rely on either mechanism here to be sure we get a version.